Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/JsonExportProviderTest.cs

```diff
@@ -0,0 +1,148 @@
+                Assignments = new List<PricelistAssignment>
+                {
+                    new PricelistAssignment {Id = "A1", PricelistId = "1"},new PricelistAssignment {Id = "A2", PricelistId = "1"},
+                }
+            };
+
+            var metadata = new ExportedTypeMetadata()
+            {
+                PropertiesInfo = new[]
+                {
+                    new ExportTypePropertyInfo() { Name = "Name"} ,
+                    new ExportTypePropertyInfo() { Name = "Prices.Id"} ,
+                    new ExportTypePropertyInfo() { Name = "Id"}
+                }
+            };
+
+            //Act
+            var filteredPricelist = SerializeAndDeserialize(metadata, priceList);
+
+            //Assets
+            Assert.NotNull(filteredPricelist);
+            Assert.Equal(2, filteredPricelist.Prices.Count);
+            Assert.Null(filteredPricelist.Assignments);
+        }
+
+        [Fact]
+        public async Task FilteringValuesTest()
+        {
+            var priceList = new Pricelist()
+            {
+                Id = "1",
+                Name = "PA1",
+                Prices = new List<Price>()
+                {
+                    new Price {Id = "P1", PricelistId = "1", List = 25, CreatedDate = DateTime.Today},
+                    new Price {Id = "P2", PricelistId = "1", List = 26, EndDate = DateTime.Today},
+                },
+                Assignments = new List<PricelistAssignment>
+                {
+                    new PricelistAssignment {Id = "A1", PricelistId = "1", Name = "name1", EndDate = DateTime.Today},
+                    new PricelistAssignment {Id = "A2", PricelistId = "1", Name = "name2", EndDate = DateTime.Today},
+                }
+            };
+            var metadata = new ExportedTypeMetadata()
+            {
+                PropertiesInfo = new[]
+                {
+                    new ExportTypePropertyInfo() { Name = "Name"} ,
+                    new ExportTypePropertyInfo() { Name = "Id"},
+                    new ExportTypePropertyInfo() { Name = "Prices.Id"} ,
+                    new ExportTypePropertyInfo() { Name = "Prices.PricelistId"} ,
+                    new ExportTypePropertyInfo() { Name = "Prices.List"} ,
+                    new ExportTypePropertyInfo() { Name = "Prices.CreatedDate"} ,
+                    new ExportTypePropertyInfo() { Name = "Assignments.Id"} ,
+                    new ExportTypePropertyInfo() { Name = "Assignments.PricelistId"} ,
+                }
+            };
+
+            //Act
+            var filteredPricelist = SerializeAndDeserialize(metadata, priceList);
+
+            //Assets
+            Assert.Equal(2, filteredPricelist.Prices.Count);
+            Assert.Equal(1, filteredPricelist.Prices.Count(x => x.List == 25));
+            Assert.Equal(1, filteredPricelist.Prices.Count(x => x.List == 26));
+            Assert.Equal(1, filteredPricelist.Prices.Count(x => x.CreatedDate == DateTime.Today));
```

Comment:
Datetime.Today should not be used, as test could be ran in the edge of days. Please use some specific date.